docs/balances/FAQs.md

```diff
@@ -8,6 +8,54 @@ tags: [Balances]
 
 Valuations provide a history of the value of a portfolio whereas balances provides an indicative value of a portfolio currently, based on the previous day's closing prices for securities and FX rates.
 
+A given day's valuation is created at the approximately same time every day, and would not be updated with transactions that are created after that time. The next day's valuation would include those transactions. The balance is continuously updated with ongoing transactions as they happen, and will therefore reflect any actions taken on the portfolio, such as deposits, orders, or bonuses.
```

Comment:
🤔 This reads to me like it is saying any transaction created after a valuation was created won't be included in the valuation, which isn't the case. I think we need to be careful with the wording to make it explicit that valuations are tied to closing price and that is why they are available T+1.